unimport/__init__.py

```diff
@@ -0,0 +1,7 @@
+__description__ = "Detect or remove unused Python imports."
+__version__ = "0.1.0"
+__author__ = (
```

Comment:
this should be a string not a tuple, "John Doe, Hakan Celik" ...

Reply:
I write like this because of I saw this style at io.

https://github.com/python/cpython/blob/master/Lib/io.py#L37

Reply:
kral o tuple degil implicit string concat
```
    "Hakan Çelik <hakancelik96@outlook.com>, ",
    "Batuhan Taşkaya <>",
    "Gökmen Görgen <>",
```
yerine
```
    "Hakan Çelik <hakancelik96@outlook.com>, "
    "Batuhan Taşkaya <>, "
    "Gökmen Görgen <>"
```
yapmalisin (disarda virgul olunca tuple oluyor)